srcpkgs/umurmur/patches/remove-crypto-mem-ctrl.patch

```diff
@@ -0,0 +1,14 @@
+From: https://git.alpinelinux.org/aports/plain/main/umurmur/remove-crypto-mem-ctrl.patch
```

Comment:
Please add a note that the patch is to fix build with OpenSSL 3

Reply:
done.